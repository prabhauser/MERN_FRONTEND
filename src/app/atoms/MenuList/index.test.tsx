import { shallow } from 'enzyme';
import ManuList from './index';

describe('MenuList', () => {
    describe('MenuList snapshot', () => {
        it('should take MenuList snapshot', () => {
            const component = shallow(<ManuList itemsArray={[]} openMenuList={() => {}} />);
            expect(component).toMatchSnapshot();
        });
    });
});
