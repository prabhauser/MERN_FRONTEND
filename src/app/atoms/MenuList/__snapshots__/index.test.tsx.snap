// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuList MenuList snapshot should take MenuList snapshot 1`] = `
<ForwardRef(Container)>
  <ForwardRef(Box)
    sx={
      Object {
        "display": Object {
          "md": "block",
          "xs": "none",
        },
        "flexGrow": 0,
      }
    }
  >
    <ForwardRef(Menu)
      anchorEl={[Function]}
      anchorOrigin={
        Object {
          "horizontal": "right",
          "vertical": "top",
        }
      }
      className="infoMenu"
      keepMounted={true}
      open={true}
      sx={
        Object {
          "display": Object {
            "md": "block",
            "xs": "none",
          },
        }
      }
      transformOrigin={
        Object {
          "horizontal": "right",
          "vertical": "top",
        }
      }
    />
  </ForwardRef(Box)>
  <ForwardRef(Box)
    sx={
      Object {
        "display": Object {
          "md": "none",
          "xs": "block",
        },
        "flexGrow": 0,
      }
    }
  >
    <ForwardRef(Menu)
      anchorEl={[Function]}
      className="infoMenu"
      open={true}
      sx={
        Object {
          "display": Object {
            "md": "none",
            "xs": "block",
          },
        }
      }
    />
  </ForwardRef(Box)>
</ForwardRef(Container)>
`;
